src/scribe_data/check/check_query_forms.py

```diff
@@ -556,30 +390,10 @@ def check_query_forms() -> None:
             )
             index += 1
 
-        # Check for unique return forms and handle the error message.
-        unique_check_result = check_unique_return_forms(query_text)
-        if unique_check_result is not True:
-            error_output += f"\n{index}. {query_file_str}: {unique_check_result}\n"
-            index += 1
-
-        if undefined_forms := check_undefined_return_forms(query_text):
-            error_output += f"\n{index}. {query_file_str}: {undefined_forms}\n"
-            index += 1
-
-        if unreturned_optional_forms := check_unreturned_optional_forms(query_text):
-            error_output += (
-                f"\n{index}. {query_file_str}: {unreturned_optional_forms}\n"
-            )
-            index += 1
-
-        if defined_unreturned_forms := check_defined_return_forms(query_text):
-            error_output += f"\n{index}. {query_file_str}: {defined_unreturned_forms}\n"
-            index += 1
-
-        # Check the order of variables in the WHERE and SELECT clauses.
-        select_where_labels_matching = check_forms_order(query_text)
-        if not select_where_labels_matching:
-            error_output += f"\n{index}. {query_file_str}:\n  - The order of variables in the SELECT statement does not match their order in the query.\n"
+        # Check the order of variables in the WHERE and SELECT clauses, and if all forms are defined and returned.
+        forms_order_and_definition_check = validate_query_forms(query_text)
+        if forms_order_and_definition_check:
+            error_output += f"\n{index}. {query_file_str}:\n  - {forms_order_and_definition_check}\n"
             index += 1
```

Comment:
Great job on removing your three functions and integrating them into the other function! But, I believe the new function name is too general. one can't  distinguish it from the file's main function `check_query_forms` . What do you think, @andrewtavis?

Reply:
I'll check this a bit later and see if any renaming is needed based on how it's reading in full :) Thanks for the check @OmarAI2003 and for the work in integrating it all, @axif0!